pa_matchbatch.sh

```diff
@@ -207,42 +210,46 @@ for currentItem in ${repoRevMap}; do
 	to_rev=$(echo -n ${currentItem} | cut -d'|' -f1)
 	repo_url=$(echo -n ${currentItem} | cut -d'|' -f2)
 	repo=$(parseRepoName ${repo_url})
-	foss_name=$(stripPrivateSuffix ${repo})
+	public_name=$(stripPrivateSuffix ${repo})
 
-	# If a non-empty value for ONLY_ON was passed-in, then we want to run
+	# If a non-empty value for ONLY_ON was passed in, then we want to run
 	# ticketmatch only on those repos. This is equivalent to ignoring all
 	# repos that aren't a part of ONLY_ON.
-	if [[ ! -z "${only_on}" ]] && ! containsElement "${only_on}" "${foss_name}"; then
-		ignored_repos="${ignored_repos} ${foss_name}"
+	if [[ ! -z "${only_on}" ]] && ! containsElement "${only_on}" "${public_name}"; then
+		ignored_repos="${ignored_repos} ${public_name}"
 	fi
 
 	# something like
-	#    [[ "${ignored_repos}" =~ ${foss_name} ]]
+	#    [[ "${ignored_repos}" =~ ${public_name} ]]
 	# will not work when $ignored_repos contains e.g. puppet-agent
-	# and $foss_name is puppet.
-	if containsElement "${ignored_repos}" "${foss_name}"; then
+	# and $public_name is puppet.
+	if containsElement "${ignored_repos}" "${public_name}"; then
 		continue
 	fi
 
-	echo "<><><><><><><><><><>"
+	print_divider
+
 	cloneOrFetch "${to_rev}" "${repo_url}"
 	pushd ${repo}
 
 	  # get current version [from_rev]
 	  from_rev=$(git describe --abbrev=0 --tags) # | sed -e 's/^v//')
-	  fix_ver=$(getFixVerFor "${foss_name}")
-	  jiraProjectId=$(getJiraProjectIdFor "${foss_name}")
-	  jiraFixedInProject=$(getJiraFixedInFor "${foss_name}")
+	  fix_ver=$(getFixVerFor "${public_name}")
+	  jiraProjectId=$(getJiraProjectIdFor "${public_name}")
+	  jiraFixedInProject=$(getJiraFixedInFor "${public_name}")
 
-	  versionsUsed="${versionsUsed}\n${foss_name}:${fix_ver}"
+	  versionsUsed="${versionsUsed}\n${public_name}:${fix_ver}"
 
-	  echo "Checking: ${foss_name}"
-	  echo "from_rev: $from_rev, to_rev: $to_rev, fix_ver: $fix_ver"
-	  ruby ${TICKETMATCH_PATH}/ticketmatch.rb --ci -f "${from_rev}" -t "${to_rev}" -p "${jiraProjectId}" -v "${jiraFixedInProject} ${fix_ver}" 2> /dev/null
+	  echo_bold "Ticketmatch results for $public_name"
+	  echo "(From tag '$from_rev' to ref '$to_rev' - JIRA fixVersion is '$(getJiraProjectIdFor $public_name) $fix_ver')"
+	  echo
+	  ruby ${TICKETMATCH_PATH}/ticketmatch.rb --ci -f "${from_rev}" -t "${to_rev}" -p "${jiraProjectId}" -v "${jiraFixedInProject} ${fix_ver}" 2> /dev/null | sed 's/^/\t/g'
 	  echo
 	popd
 done
 
-echo "<><><><><><><><><><>"
-echo "Versions used for JIRA searches (foss_repo:version)"
-printf "%b\n" ${versionsUsed}
+print_divider
+
+echo "The following versions were used for JIRA searches (repo_name:version)"
+echo_bold "If these versions are incorrect, you should create a version overrides file and try again. See the README."
```

Comment:
It's easy to miss incorrect automatically determined fixVersions in the output (this happened initially for 6.3.0), so I've put a bold note describing what to do if they're wrong here.